File: zendrive-ui/src/api/models/user/UpdateUserDto.ts
export class UpdateUserDto {
	firstName: string;
	lastName: string;
	displayName: string;
	email: string;

	constructor(firstName: string, lastName: string, displayName: string, email: string) {
		this.firstName = firstName;
		this.lastName = lastName;
		this.displayName = displayName;
		this.email = email;
	}
}

File: zendrive-api/src/main/java/com/zendrive/api/rest/models/dto/user/UpdateUserDto.java
package com.zendrive.api.rest.models.dto.user;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotEmpty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UpdateUserDto {
	@NotEmpty
	private String firstName;

	@NotEmpty
	private String lastName;

	@NotEmpty
	private String displayName;

	@Email
	@NotEmpty
	private String email;
}
